platform/ios/src/MGLMapView.mm

```diff
@@ -1421,10 +1418,47 @@ - (void)actionSheet:(UIActionSheet *)actionSheet didDismissWithButtonIndex:(NSIn
         [[UIApplication sharedApplication] openURL:
          [NSURL URLWithString:feedbackURL]];
     }
-    // skips to 4 because button is conditionally added after cancel (index 3)
-    else if (buttonIndex == actionSheet.firstOtherButtonIndex + 4)
+    else if (buttonIndex == actionSheet.firstOtherButtonIndex + 3)
+    {
+        NSString *messageTense;
+        NSString *participate;
+        NSString *optOut;
+
+        if ([[NSUserDefaults standardUserDefaults] boolForKey:@"MGLMapboxMetricsEnabled"])
+        {
+            messageTense = @"are helping to";
```

Comment:
I know we haven’t revisited localization yet, but splitting strings like this complicates localization. The whole sentence should be kept together in one string.